### DISCUSSION TITLE
What does the letter next to the file's date mean?

### DISCUSSION THREAD

**Discussion by Ailothaen** [d2phap/ImageGlass#957] [Q&A] [answered]
Hello everyone,

Since a recent update, I can now notice a letter next to the file's date in the window title. I already encountered **(m)** and **(o)** :

![20210106-234750-001753](https://user-images.githubusercontent.com/11133116/103827702-e1884080-5079-11eb-95d7-2415ed0890f6.png)

![20210106-234757-001754](https://user-images.githubusercontent.com/11133116/103827705-e220d700-5079-11eb-8eaf-b33c5d9dfd7e.png)

What do they mean? Are there other status possible?

**Comment by fire-eggs** [comment, marked as answer]
Exif.DateTimeOriginal (o), 
Exif.DateTime, or 
File.LastWriteTime (m)

In that order. E.g. file has no Exif data, shows the file's last modified time.


------------
Added by Phap:
The details is here https://github.com/d2phap/ImageGlass/issues/687#issuecomment-603925112
